app/views/admin/summary.erb

```diff
@@ -0,0 +1,111 @@
+<% content_for :breadcrumbs do %>
+  <%= render "govuk_publishing_components/components/breadcrumbs", {
+  collapse_on_mobile: true,
+    breadcrumbs: [
+      {
+        title: "All finders",
+        url: root_path
+      },
+      {
+        title: "#{current_format.title} finder",
+        url: "/admin/#{current_format.admin_slug}"
+      }
+    ]
+  } %>
+<% end %>
+<% content_for :page_title, "Edit #{current_format.title} finder" %>
+<% content_for :title, "#{current_format.title} finder" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    <%= render "govuk_publishing_components/components/inset_text", {
+    } do %>
+      <%= link_to("View on website (opens in new tab)", current_format.live_url, class: "govuk-link", target: "_blank", rel: "noopener") %>
+    <% end %>
+
+    <%= render "govuk_publishing_components/components/secondary_navigation", {
+      id: "nav_1234",
+      aria_label: "Document navigation",
+      items: [
+        {
+          label: "Finder documents",
+          href: documents_path(params[:document_type_slug]),
+        },
+        {
+          label: "Finder setup",
+          href: "/admin/#{current_format.admin_slug}",
+          current: true,
+        },
```

Comment:
Nice work! Just to double-check, we're not intending to show this tabbed view on the 'main' finder page yet, right? (...until we pick up https://trello.com/c/JoK7tUbQ/443-release-the-edit-finder-forms-feature, so as not to point publishers to forms that aren't ready to use yet).

Reply:
Yeah, no plans to add the tabs to the main finder page until this is ready for publishers to use